style.css

```diff
@@ -1,3 +1,83 @@
 #root {
-  color: red;
+  max-width: 900px;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  flex-wrap: wrap;
 }
+body{
+  background-color: #F4ECE6;
+}
+
+
+* {
+  box-sizing: border-box;
+}
+
+.episode-title{
+  box-shadow: 3px 3px #1D0029;
+  border-radius: 5px;
+  padding: 5px;
+  background-color: #1D0029;
+  color: whitesmoke;
+ 
+} 
+.episode-title:hover{
+  text-decoration: underline;
```

Comment:
This made me think it was clickable! Maybe make it so on a click it only renders that title to the screen? Or takes us to a different page?